enterprise/backend/test/metabase_enterprise/serialization/v2/extract_test.clj

```diff
@@ -1496,14 +1497,23 @@
                                       clickdash-eid :entity_id} {:name          "Dashboard with click behavior"
                                                                  :collection_id coll5-id
                                                                  :creator_id    mark-id}
-                       DashboardCard _                          {:card_id c3-1-id
+                       DashboardCard _                          {:card_id      c3-1-id
                                                                  :dashboard_id clickdash-id
                                                                  :visualization_settings
                                                                  ;; Top-level click behavior for the card.
-                                                                 {:click_behavior {:type "link"
-                                                                                   :linkType "question"
-                                                                                   :targetId c3-2-id
-                                                                                   :parameterMappings {}}}}
+                                                                 (let [dimension  [:dimension [:field "something" {:base-type "type/Text"}]]
+                                                                       mapping-id (json/generate-string dimension)]
+                                                                   {:click_behavior {:type     "link"
+                                                                                     :linkType "question"
+                                                                                     :targetId c3-2-id
+                                                                                     :parameterMapping
```

Comment:
I noticed this was wrong before, probably because the docstrings of `import-viz-click-behavior-mappings` and `export-viz-click-behavior-mappings` have it wrong too. Can you clean those up with this fix?